docs/vwii/vwii-modding.md

```diff
@@ -29,70 +27,53 @@ We will now place the required Homebrew files on the SD Card.
 1. Copy the `compat_installer.elf` file to the `wiiu/apps` folder on the root of your SD Card.

```

Comment:
Wouldn't it make sense to use the rpx port which works on both Aroma *and* Tiramisu? `.elf` is not supported on Aroma at all.